selfdrive/ui/ui.cc

```diff
@@ -202,6 +202,8 @@ void UIState::updateStatus() {
       status = STATUS_WARNING;
     } else if (alert_status == cereal::ControlsState::AlertStatus::CRITICAL) {
       status = STATUS_ALERT;
+    } else if (alert_status == cereal::ControlsState::AlertStatus::OVERRIDE) {
```

Comment:
Should we check the OpenpilotState instead like below instead? I could see it either way